addon/search/jump-to-line.js

```diff
@@ -42,10 +36,9 @@
 
       var match;
       if (match = /^\s*([\+\-]?\d+)\s*\:\s*(\d+)\s*$/.exec(posStr)) {
-        cm.setCursor(Pos(interpretLine(cm, match[1]), Number(match[2])))
```

Comment:
Below `setCursor` doesn't use `Pos`, so it's more consistent to do the same here.